openedx/features/edly_panel/api/v1/views.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+
+@api_view(['GET'])
+def get_maus(request):
+    """
+        For the time being just returning dummy data.
```

Comment:
nit: remove these extra indent
Also change the description:
```
Fetch the data related to monthly active users.
```

Reply:
done.